public/app/features/alerting/unified/components/notification-policies/NotificationPoliciesList.tsx

```diff
@@ -118,14 +66,14 @@ const AmRoutes = () => {
 
   const { currentData: alertGroups, refetch: refetchAlertGroups } = useGetAlertmanagerAlertGroupsQuery(
     { amSourceName: selectedAlertmanager ?? '' },
-    { skip: !canSeePoliciesTab || !canSeeAlertGroups || !selectedAlertmanager }
```

Comment:
We don't need to check these kinds of things, as we now can assume that if we're rendering this, then the decision of whether or not they can see the policies has been made elsewhere